src/main/resources/i18n/Validation_nl.properties

```diff
@@ -0,0 +1,99 @@
+#
+# This resource bundle is used for the query validation code to produce i18n messages
+#
+# The keys have the format of rule class name and then message type within that.  Most rules
+# will only have 1 or 2 message keys
+#
+# Please try and keep this sorted within rule class and use # between sections so the IDEA Ctrl-Alt-L reformat does not bunch
+# them too tightly.
+#
+# REMEMBER - a single quote ' in MessageFormat means things that are never replaced within them
+# so use 2 '' characters to make it one ' on output.  This will take for the form ''{0}''
+#
+ExecutableDefinitions.notExecutableType=Validatiefout ({0}) : Type definitie ''{1}'' is niet uitvoerbaar
+ExecutableDefinitions.notExecutableSchema=Validatiefout ({0}) : Schema definitie is niet uitvoerbaar
+ExecutableDefinitions.notExecutableDirective=Validatiefout ({0}) : Directive defnitie ''{1}'' is niet uitvoerbaar
+ExecutableDefinitions.notExecutableDefinition=Validatiefout ({0}) : Aangeleverde definition is niet uitvoerbaar
+#
+FieldsOnCorrectType.unknownField=Validatiefout ({0}) : Veld ''{1}'' in type ''{2}'' is ongedefinieerd
+#
+FragmentsOnCompositeType.invalidInlineTypeCondition=Validatiefout ({0}) : Inline fragment type condition is ongeldig, moet op een Object/Interface/Union zitten
+FragmentsOnCompositeType.invalidFragmentTypeCondition=Validatiefout ({0}) : Fragment type condition is ongeldig, moet op een Object/Interface/Union zitten
+#
+KnownArgumentNames.unknownDirectiveArg=Validatiefout ({0}) : Onbekende directive argument ''{1}''
+KnownArgumentNames.unknownFieldArg=Validatiefout ({0}) : Onbekende field argument ''{1}''
+#
+KnownDirectives.unknownDirective=Validatiefout ({0}) : Onbekende directive ''{1}''
+KnownDirectives.directiveNotAllowed=Validatiefout ({0}) : Directive ''{1}'' is hier niet toegestaan
+#
+KnownFragmentNames.undefinedFragment=Validatiefout ({0}) : Ongedefinieerd fragment ''{1}''
+#
+KnownTypeNames.unknownType=Validatiefout ({0}) : Ongeldig type ''{1}''
+#
+LoneAnonymousOperation.withOthers=Validatiefout ({0}) : Anonieme operation met andere operations
+LoneAnonymousOperation.namedOperation=Validatiefout ({0}) : Operation ''{1}'' volgt een anonieme operatie
```

Comment:
Ik twijfel zwaar of ik "operation" naar "operatie" moet veranderen. Naar mijn mening past het niet. De Duitsers hebben dit probleem niet :,)

Reply:
Is de graphql spec niet vertaald? Misschien kun je de Nederlandse definitie daarvan overnemen?